### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by Roomsycakes01**
Jake Rooms: Hi here's my portfolio code for trainer review.

### CODE REVIEW

File: portfolio.css [hunk 1]
```diff
@@ -0,0 +1,216 @@
+*{
+	box-sizing: border-box;
+	color: #fff;
+}
+
+/* Background image css */
+
+body{
+	background-image: url(mountain_bg.jpg);
+	background-repeat: no-repeat;
```
Comment (charliecog) [inline portfolio.css:10]: not sure this is doing anything

File: portfolio.css [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+	background-position: bottom;
+	background-color: #FBEEC1;
+	background-attachment: fixed;
```
Comment (charliecog) [inline portfolio.css]: not sure this is doing anything

File: portfolio.css [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+.top_link{
+	position: fixed;
+	top: 2vh;
+}
+
+/*formatting on the left, right and center text sections and initial home section*/
+
+.section_home{
+	width: 80%;
+	margin: 0 auto;
```
Comment (charliecog) [inline portfolio.css]: not sure this is doing anything

File: portfolio.css [hunk 4]
```diff
@@ -0,0 +1,216 @@
+*{
```
Comment (charliecog) [inline portfolio.css]: formatting wise please can you update this file to have a space between your css selector and the `{`

Comment (charliecog) [reply portfolio.css]: eg: `* {`

File: portfolio_home.html [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+					<a href='portfolio_project3.html'>Project 3</a>
+					<a href='portfolio_project4.html'>Project 4</a>
+				</nav>
+			</div>
+		</section>
+		<!--Html for the footer , contact details and social media buttons-->	
+		<footer class='link_bar'>
+			<!--Html for the contact information-->
+			<div class='contact_info'>
+				<h2>Contact me</h2>
```
Comment (charliecog) [inline portfolio_home.html]: this is styled up to look like a button but doesnt have the functionality. Best to make it look more like a title than a button

File: portfolio_home.html [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+		<!--Html for the top navigation bar-->
+		<nav class='link_bar'>
+			<a class= 'top_link' href='portfolio_aboutme.html'>About me</a>
+		</nav>
+		<!--Html for the image and project navigation bar-->
+		<section class='section_home'>
+			<img class='img_home' src='jake_face.jpg' alt='my face'/>
+			<!--Html for the brojects navigation menu-->
+			<div class='projects'>
+				<h2>Projects</h2>
```
Comment (charliecog) [inline portfolio_home.html]: this is styled up to look like a button but doesnt have the functionality. Best to make it look more like a title than a button

File: portfolio_home.html [hunk 7]
```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang='en'>
+<head>
+    <meta charset='UTF-8'/>
+	<meta name='viewport' content='width=device-width, initial-scale=1.0'/>
+    <title>Jake Rooms - Portfolio</title>
+	<link rel='stylesheet' href='normalize.css'/>
+	<link rel='stylesheet' href='portfolio.css'/>
```
Comment (charliecog) [inline portfolio_home.html]: always link your custom css styles as the last one in the list, this way you can override styles that have a similar precedence. (please check this across all html files)

**Review by charliecog — COMMENTED** [review]
Looking good, almost there! 

general comments:
Filenames - edit them to be slightly more url friendly. Make portfolio_home.html into index.html, and remove the portfolio_ prefix from all the others. 

There is a school of thought that says to avoid placing comments in HTML. This is opinion so up to you if you want to. The thought process is that HTML should be written in a self explanatory way.

File: portfolio.css (hunk 1 above)
Comment (Roomsycakes01) [reply portfolio.css:10]: On smaller screen sizes without this the code tiles a second copy of the image below

File: portfolio_home.html (hunk 5 above)
Comment (Roomsycakes01) [reply portfolio_home.html]: lowered the border radius and removed the outlines

File: portfolio_home.html (hunk 6 above)
Comment (Roomsycakes01) [reply portfolio_home.html]: lowered the border radius and removed the outlines

**Review by charliecog — APPROVED** [review]

**Pull request merged**